thetis/coupled_timeintegrator_2d.py

```diff
@@ -14,12 +14,10 @@ class CoupledTimeIntegrator2D(timeintegrator.TimeIntegratorBase):
     """
     __metaclass__ = ABCMeta
 
-    @abstractproperty
     def swe_integrator(self):
         """time integrator for the shallow water equations"""
         pass
 
-    @abstractproperty
```

Comment:
This was required for the way I have implemented `CoupledMatchingTimeIntegrator2D`, but there might be another way.